charters/PM/charter.md

```diff
@@ -0,0 +1,64 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
+    * Focuses the DCP community on a documented set of high-level customer priorities
+    * Excites users about the future of DCP
+
+* **Request for Comments (RFC)** is the transparent process that manages all significant enhancements to the DCP software technical and governance documentation, both normative and informative
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
+
+*Communication*
```

Comment:
I've been thinking about how to capture the current state of the DCP. I really like the proposals in this charter for representing how improvements will be made going forward, but I think there's a gap around where we're at right now. For example, if I am new to the project I really want a single concise summary of what the DCP can do _right now_. 

As a strawman proposal, how about the PM team take on the responsibility of publishing an annual "State of the DCP Nation" report? I'd imagine this containing a summary of the current DCP - both in terms of features and design - and maybe the key updates to the thematic roadmap. It could also contain a selection of key user findings over the last year that can guide the overall vision for the following year

Reply:
I think a blog post or bioarchiv pre-print rather than something which gets "officially" published. That being said we should start considering to target the NAR database issue for bi-annual publications